pkgs/apps/meep/default.nix

```diff
@@ -0,0 +1,103 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, autoreconfHook, pkg-config
+, gfortran, mpi, blas, lapack, fftw, hdf5-full, swig, gsl, harminv, libctl
+, libGDSII, openssh, guile
+# Python
+, python, numpy, scipy, matplotlib, h5py, cython, autograd, mpi4py
+} :
+
+buildPythonPackage rec {
+  pname = "meep";
+  version = "1.18.0";
+
+  src = fetchFromGitHub {
+    owner = "NanoComp";
+    repo = pname;
+    rev = "v${version}";
+    sha256= "08l4mczkh08dp90c4dlnyx6lsg093li0xqnnbh7q8k363cr8lx81";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    gfortran
+    pkg-config
+    swig
+    mpi
+  ];
+
+  buildInputs = [
+    gsl
+    blas
+    lapack
+    fftw
+    hdf5-full
+    harminv
+    libctl
+    libGDSII
+    guile
+    gsl
+  ];
+
+  propagatedBuildInputs = [
+    mpi
+    numpy
+    scipy
+    matplotlib
+    h5py
+    cython
+    autograd
+    mpi4py
+  ];
+
+  propagatedUserEnvPkgs = [ mpi ];
+
+  dontUseSetuptoolsBuild = true;
+  dontUsePipInstall = true;
+  dontUseSetuptoolsCheck = true;
+
+  HDF5_MPI = "ON";
+  PYTHON = "${python}/bin/${python.executable}";
+
+  enableParallelBuilding = true;
+
+  configureFlags = [
+    "--without-libctl"
```

Comment:
Is that intended? `libctl` is in the `buildInputs`.

Reply:
Yep, it is still needed and wont work without. So one part of `configure` finds it, while another one is not finding it ...